### PR TITLE
[TM FIRST] Removes D class

### PR DESCRIPTION
**Pull request by Kitsunemitsu**
## About the Pull Request
This sets D class Slots to 0

## Why It's Good For The Game

People always complain about it.
I want to prove a point

## Changelog

:cl:
del: Removed D class
/:cl:

### CODE REVIEW

**Comment by CerberusHellHound** [comment]
die

File: maps/site53/job/jobs/misc.dm [hunk 1]
```diff
@@ -3,11 +3,11 @@
 	department = "Civilian"
 	selection_color = "#E55700"
 	economic_power = 1
-	total_positions = 999
-	spawn_positions = 999
+	total_positions = 0
```
Comment (CerberusHellHound) [inline maps/site53/job/jobs/misc.dm:6]: 100000000000

File: maps/site53/job/jobs/misc.dm [hunk 2]
```diff
@@ -3,11 +3,11 @@
...
+	spawn_positions = 0
```
Comment (CerberusHellHound) [inline maps/site53/job/jobs/misc.dm:7]: 100000000000

**Review by CerberusHellHound — COMMENTED** [review]
add more than a gazzilion spots

**Comment by Vizoplez** [comment]
D-class is our designated overflow job, removing it without providing an alternative is asking for issues.

**Comment by Kitsunemitsu** [comment]
We actually have Office workers now to be our overflow job.
In all I think that this would be a positive RP change for the server

**Pull request closed without merging**